input/fsh/questionnaires/MeaslesHealthCheckQuestionnaire.fsh

```diff
@@ -0,0 +1,191 @@
+Alias: $usage-context-type = http://terminology.hl7.org/CodeSystem/usage-context-type
+Alias: $sct = http://snomed.info/sct
+
+Instance: MeaslesHealthCheckQuestionnaire
+InstanceOf: Questionnaire
+Usage: #example
+* url = "https://build.fhir.org/ig/tewhatuora/cinc-fhir-ig/Questionnaire/MeaslesHealthCheckQuestionnaire"
```

Comment:
better FSH syntax for readability and more resilient to IG URL changes? 
` * url = Canonical(MeaslesHealthCheckQuestionnaire)`